planetarium/NineChronicles
# Feature/new arena score system

### Description

In the past, defender in arena didn't get any score.
This is a new scoring system of defender.

### How to test

1. Try to battle in arena.
2. Re-load ranking board.
2. Check score of defender.

### Related Links

[아레나 스코어 로직 수정 (진사람 점수 까기)](https://app.asana.com/0/958521740385861/1201406778349280/f)
[2021-11-22 9C 주간](https://www.notion.so/planetarium/2021-11-22-9C-7a0c525e580c4e87bbd2d904da6a26bf#acc7d16643c44d84950bfd58d30cdf29)
[lib9c - Feature/new arena scoring system #749](https://github.com/planetarium/lib9c/pull/749)

### Required Reviewers

@planetarium/9c-dev @Namyujeong @plafina2
